### PR TITLE
fix: Indicator theme token usage in styles

### LINKED ISSUE #840: The `Indicator` component references theme tokens for styling.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.2.6

### Description

Yamada UI allows you to use components without applying a theme. It is a mistake to use theme tokens inside components.

`packages/components/indicator/src/indicator.tsx`

```tsx
<ui.div
  className='ui-indicator__icon__ping'
  __css={{
    position: 'absolute',
    boxSize: 'full',
    rounded: 'full',
    opacity: 0.75,
    zIndex: -1,
    bg: pingColor,
  }}
  animation={animation}
/>
```

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Exerea**
Hi!
Thanks for posting on Twitter!
I'm happy to try to contribute to this OSS for the first time.
We hope Yamada UI will be a success.

Closes #840

## Description

I fixed a bug where theme tokens were used in components.
I replace from "full" to  "100%".

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by setodeve** [comment]
@Exerea 
Hi! Thank you for contributing!
Please create changeset file with executing `pnpm changeset`.

Please refer the following material.
image
https://github.com/hirotomoyamada/yamada-ui/blob/main/CONTRIBUTING.md

File: packages/components/indicator/src/indicator.tsx [hunk 1]
```diff
@@ -251,8 +251,8 @@ export const Indicator = forwardRef<IndicatorProps, "div">((props, ref) => {
               className="ui-indicator__icon__ping"
               __css={{
                 position: "absolute",
-                boxSize: "full",
-                rounded: "full",
+                boxSize: "100%",
+                rounded: "100%",
```
Comment (hirotomoyamada) [inline packages/components/indicator/src/indicator.tsx:255]:
```diff
- rounded: "100%",
+ rounded: "9999px",
```

File: .changeset/rich-suns-rescue.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/indicator": minor
```
Comment (setodeve) [inline .changeset/rich-suns-rescue.md:2]: Please edit here with a patch, not a minor.

File: pnpm-lock.yaml
Comment (setodeve) [inline pnpm-lock.yaml:1]: Please delete this file.

**Review by setodeve — CHANGES_REQUESTED** [review]

**Comment by setodeve** [comment]
@Exerea 
Could you reopen this PR if possible?

**Pull request closed without merging**